scripts/onboarding/templates/arc-k8s-extension-provisionedcluster-msi-auth/existingClusterOnboarding.json

```diff
@@ -122,9 +122,10 @@
                                         ],
                                         "extensionSettings": {
                                           "dataCollectionSettings" : {
-                                            "interval": "[parameters('dataCollectionInterval')]",
+                                             "interval": "[parameters('dataCollectionInterval')]",
                                              "namespaceFilteringMode": "[parameters('namespaceFilteringModeForDataCollection')]",
-                                             "namespaces": "[parameters('namespacesForDataCollection')]"
+                                             "namespaces": "[parameters('namespacesForDataCollection')]",

```

Comment:
same as the others, should this change be part of syslogandCI template as well?

Reply:
same reason